centos8/http/centos8.ks

```diff
@@ -34,6 +34,8 @@ done
 
 rm -f /etc/sysconfig/network-scripts/ifcfg-[^lo]*
 
+sed -i -e 's/^\(GRUB_ENABLE_BLSCFG=\).*/\1false/g' /etc/default/grub
```

Comment:
Which GRUB menu items are missing for you? I have CentOS8, CentOS8 rescue, and system setup by default. Changing this setting doesn't give any more options.

Reply:
We have the same, but for me, and all the reproducers I did with the customer, we didn't get the any of the kernel command line options added to any of the menu items. Setting this to false solved it for me and the customer.

Reply:
[CentOS 8 actually changed how you set kernel command line options.](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_monitoring_and_updating_the_kernel/configuring-kernel-command-line-parameters_managing-monitoring-and-updating-the-kernel) They now need to be set with `grubby --update-kernel=ALL --args="<NEW_PARAMETER>"` as the settings are stored in the binary file `/boot/grub2/grubenv` not `/etc/default/grub`